paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -475,8 +475,7 @@ public class CoreOptions implements Serializable {
             key("write-buffer-spillable")
                     .booleanType()
                     .noDefaultValue()
-                    .withDescription(
-                            "Whether the write buffer can be spillable. Enabled by default when using object storage.");
```

Comment:
Don't remove this

Reply:
When we introduced external path, we don't know the underlying write path is an object store or not in FileIO, 
so here I removed the "Enabled by default when using object storage." description.

Reply:
External path is a mode, but not a default mode.

We can keep this as it is.

Reply:
Thanks, and I have remoded this and using warehouse path to check this is object or not